commit 59aba531c206077f4fd63095f8d13ba7164f9404
Author: Alexei Starovoitov <ast@fb.com>
Date:   Wed Dec 7 10:57:59 2016 -0800

    bpf: fix state equivalence
    
    [ Upstream commit d2a4dd37f6b41fbcad76efbf63124eb3126c66fe ]
    
    Commmits 57a09bf0a416 ("bpf: Detect identical PTR_TO_MAP_VALUE_OR_NULL registers")
    and 484611357c19 ("bpf: allow access into map value arrays") by themselves
    are correct, but in combination they make state equivalence ignore 'id' field
    of the register state which can lead to accepting invalid program.
    
    Fixes: 57a09bf0a416 ("bpf: Detect identical PTR_TO_MAP_VALUE_OR_NULL registers")
    Fixes: 484611357c19 ("bpf: allow access into map value arrays")
    Signed-off-by: Alexei Starovoitov <ast@kernel.org>
    Acked-by: Daniel Borkmann <daniel@iogearbox.net>
    Acked-by: Thomas Graf <tgraf@suug.ch>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1409_59aba531c206077f4fd63095f8d13ba7164f9404.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1409_59aba531c206077f4fd63095f8d13ba7164f9404.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
